### PR TITLE
Naming

### PR DESCRIPTION
**Pull request by kostja**
Consistent naming for scylla-tools-java, and use scylla-tools-java environment variable in get_stress_bin()

### CODE REVIEW

File: ccmlib/cmds/cluster_cmds.py [hunk 1]
```diff
@@ -188,8 +188,8 @@ def validate(self, parser, options, args):
 
         if options.scylla_core_package_uri:
             os.environ['SCYLLA_CORE_PACKAGE'] = options.scylla_core_package_uri
-        if options.scylla_java_tools_package_uri:
-            os.environ['SCYLLA_JAVA_TOOLS_PACKAGE'] = options.scylla_java_tools_package_uri
+        if options.scylla_tools_java_package_uri:
+            os.environ['SCYLLA_TOOLS_JAVA_PACKAGE'] = options.scylla_tools_java_package_uri
```
Comment (fruch) [inline ccmlib/cmds/cluster_cmds.py]: This environment variable is using in at least 6 other repos, can't merge this without being backward compatible.

Comment (kostja) [reply ccmlib/cmds/cluster_cmds.py]: Why not fix all of them? It's a trivial change. Backward compatibility is technical debt. There is already 4 ways to provide tools-java package, and backward compatibility will add the fifth one. I can keep the old variable, but this upsets the purpose of the patch. The goal of the patch is to make the code easier to follow, not harder.

Comment (kostja) [reply ccmlib/cmds/cluster_cmds.py]: For dtest, I made a matching pull request to fix.
And as I have been suggesting earlier, if scylla-ccm was a submodule, there would have been no issue.

Comment (fruch) [reply ccmlib/cmds/cluster_cmds.py]: * https://github.com/scylladb/scylla-pkg/blob/a196f0b3187b95602fb606200f837308699549d5/scripts/jenkins-pipelines/groovy_modules/test.groovy#L136
* https://github.com/scylladb/scylla-java-driver-matrix/blob/0eb756c877fe9047de3fb27090f948854bfe948c/scripts/run_test.sh#L100
* https://github.com/scylladb/python-driver-matrix/blob/master/scripts/run_test.sh#L86
* https://github.com/scylladb/scylla-cassandra-unit-tests/blob/master/scripts/run_test.sh#L106

Comment (fruch) [reply ccmlib/cmds/cluster_cmds.py]: The main issue is that we need to do all of them in sync, hence why I prefer it to be backward compatible first, so we can merge the ccm change, and then the others.
and when all are done, remove the unneeded code.

**Pull request closed without merging**